auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailFragment.java

```diff
@@ -86,6 +92,16 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         }
     }
 
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        if (!mPasswordInput.isHintAnimationEnabled()) {
```

Comment:
You also used this in some activity. If it was enabled in XML it would start its animation during the view creation.